src/routes.js

```diff
@@ -39,11 +39,17 @@ import StandalonePostEditor from './containers/StandalonePostEditor'
 import Admin from './containers/Admin'
 import PageWithNav from './containers/PageWithNav'
 import TestBench from './containers/TestBench'
+import socketIOClient from 'socket.io-client'
+import sailsIOClient from 'sails.io.js'
 import { debug } from './util/logging'
 import { makeUrl } from './util/navigation'
 import { get, isEmpty, pick } from 'lodash'
 import config from './config'
 
+let io = sailsIOClient(socketIOClient)
```

Comment:
Ok, great! The reason why I got this code up online as early as possible in my process was to make sure I was doing it in a most suitable / clean way :) thanks Lawrence.